infra/experimental/sanitizers/ExecSan/README.md

```diff
@@ -1,21 +1,32 @@
 # Shell Injection Detection with `ptrace`
 
+We use `ptrace` to instrument system calls made by the target program to detect if our `sand` command was injected into the *shell* of the target `oyster` program during execution and led to the production of a `pearl` file.
+Our instrumentation verifies the existence of `/tmp/pearl` after every `execve` and each `wait4` after `clone`, which proves the existence of shell injection vulnerabilities.
+
```

Comment:
Added a TODO for this, I will try to figure out how to [collect the PIDs of the target's children](https://man7.org/linux/man-pages/man2/ptrace.2.html#:~:text=*%20%20The%20execing%20tracee,leader%27s%20thread%20ID.) so that the tracer knows which files to look for : )